docs/section6/Rule6-12.md

```diff
@@ -0,0 +1,36 @@
+
+# Lighting - Rule 6-12
+
+**Rule ID:** 6-12  
+**Rule Description:** Proposed building is modeled with daylighting controls  
+**Appendix G Section:** Section 6 Lighting  
+**Appendix G Section Reference:** Section G3.1-6(h) Lighting: Modeling Requirements for the Proposed design  
+
```

Comment:
1. The purpose of this rule appears to be verifying that mandatory lighting controls are specified. If so, I beleive it should be applied to U-RMR, not P-RMR.
2. I think we also need a rule that U-RMR = P-RMR.

Reply:
I beleive we agreed that this rule will apply to U-RMR and not P-RMR (#1 above). So the RDS looks good to me except "p" should be replaced with "u" throughout. E.g., in line #16 it should say For zone_u in U_RMR...zones:`

Reply:
Corrected.